shared/src/commonMain/kotlin/db/repository/user/RemoteUserRepository.kt

```diff
@@ -38,16 +33,22 @@ class RemoteUserRepository(private val httpClient: HttpClient) :
         }
     }
 
-    override suspend fun fetchById(id: String) {
-        database.getById(id)
+    override suspend fun fetchById(id: String): User {
```

Comment:
I think you're missing something here. These functions were created to query te local database if i'm not mistaken. You should create extra functions to get the firebase data and store them in your local database

Reply:
I think the naming of the function is just bad, in the onboarding we used find to define a function that queries the local database and get to define a function that queries the api, in our case firebase. Instead of get I used fetch.